src/css/yasr.custom.css

```diff
@@ -98,15 +98,16 @@
 }
 
 .yasr .uri-cell {
-  display: inline;
-  word-break: normal !important;
-  word-wrap: normal !important;
+    display: inline;
+    word-break: normal !important;
+    word-wrap: break-word !important;
```

Comment:
I don't get this. What's the point in using both `word-break: normal` which dosn't split lenghty words when they overflow the box boundaries and `word-wrap: break-word` which BTW should be replaced with `overflow-wrap`? IMO we should leave only the `overflow-wrap: break-word` here. And same for the line 106 below.

Reply:
I will change `word-wrap` with `overflow-wrap`. `word-break` is set to normal because it has value break-all from original yasr css style.